### PR TITLE
Fix action-context tests

### PR DESCRIPTION
**Pull request by imalberto**

### CODE REVIEW

File: lib/tests/autoload/app/autoload/action-context-tests.common.js [hunk 1]
```diff
@@ -225,12 +231,14 @@ YUI.add('mojito-action-context-tests', function(Y, NAME) {
                         return "routes";
                     }
                 }
+                // models: {},
```
Comment (drewfish) [inline lib/tests/autoload/app/autoload/action-context-tests.common.js]: Why was this added?

Comment (imalberto) [inline lib/tests/autoload/app/autoload/action-context-tests.common.js]: This was not added, but commented out from the original tests. But can be safely removed.

File: lib/app/autoload/action-context.common.js [hunk 2]
```diff
@@ -224,7 +225,7 @@ YUI.add('mojito-action-context', function(Y, NAME) {
 
         for (addonName in dependencies) {
             if (dependencies.hasOwnProperty(addonName)) {
-                if (mods.indexOf(addonName + '-addon') > -1 || addonName === 'core') {
+                if (mods.indexOf(addonName) > -1 || addonName === 'core') {
```
Comment (drewfish) [inline lib/app/autoload/action-context.common.js:228]: Are you sure that this is safe to do? (Yeah, I know this code will be going away.) You might instead try changing the list ['first', 'second', ...] below to ['first-addon', 'second-addon', ...].

Comment (imalberto) [inline lib/app/autoload/action-context.common.js:228]: I did that initially, but then thought of "search-app" that does not use that convention for their addons. If the norm is "xxx-addon", sure we can proceed the original way.

Comment (drewfish) [inline lib/app/autoload/action-context.common.js:228]: This change seems dangerous to me, though I'm still trying to work out how.  However, since this code will be going away (soon?), let's just ship it.

**Comment by drewfish** [comment]
+1
